### PR TITLE
Add May General Meeting mins

### PR DESCRIPTION
**Pull request by rnjudge**
Add meeeting minutes from the May general meeting.

### CODE REVIEW

File: general/2024/2024-05-02.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+## Administrative
+- Meeting run by Rose Judge
+- Approve meeting minutes from April meeting: https://github.com/spdx/meetings/pull/676
+  - Approved
+
+## Presentation: State of SBOM in 2024
+- Presenter: Dr. Allan Friedman
+- https://www.cisa.gov/sbom
+- SBOM is now reasonable understod but still variance in implementation and guidance (esp. tooling)
```
Comment (kestewart) [inline general/2024/2024-05-02.md:11]:
```suggestion
- SBOM is now reasonable understood but still variance in implementation and guidance (esp. tooling)
```

Comment (rnjudge) [reply general/2024/2024-05-02.md]: Fixed

**Comment by rnjudge** [comment]
Approved in the June General Meeting. Merging.

**Comment by Sintayew4** [comment]
#681

**Pull request merged**